schemas/com.snowplowanalytics.snowplow.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,303 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a bad row resulting from schema violations",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "schema_violations",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the schema violations",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "schemaKey": {
+          "type": "string",
+          "description": "Schema which led to the issue",
+          "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+        },
+        "messages": {
+          "description": "Schema violations",
+          "anyOf": [
```

Comment:
These schema violation error types are brilliant Ben: so specific - and I think, collectively, exhaustive.